extensions/amp-inline-gallery/1.0/amp-inline-gallery-pagination.css

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-inline-gallery-pagination[inset] {
+  display: block !important;
+  height: 20px; /* Default height. */
+  width: 100%;
```

Comment:
Is this actually desirable?

Reply:
Yes, otherwise it has a width of `0` and looks like this: 
![image](https://user-images.githubusercontent.com/10456171/103011061-4cc50500-4507-11eb-89be-d13054883da6.png)